route/build/src/base_route.f90

```diff
@@ -0,0 +1,60 @@
+MODULE base_route
+
+  ! Abstract class for routing method
+
+  implicit none
+
+  private
+  public:: base_route_rch
```

Comment:
I'd add 1-line comments after each declaration

Reply:
adding one comment for this and below